client/css/style.css

```diff
@@ -678,9 +678,15 @@ button, .user {
 }
 #chat .part .from:before,
 #chat .quit .from:before {
-    font-family: FontAwesome;
+	font-family: FontAwesome;
 	content: "\f08b";
 	color: #FF4136;
+	display: inline-block;
+	-webkit-transform: rotate(180deg);
+	-moz-transform: rotate(180deg);
+	-ms-transform: rotate(180deg);
+	-o-transform: rotate(180deg);
+	transform: rotate(180deg);
```

Comment:
That would be more consistent with what we use in other parts of the CSS (that would break anyway with IE<10). 

BTW, it's not to be done here, but the -webkit-transform hanging around in the CSS should IMHO be removed, if there is a "special needs" browser which requires to use prefixed property for transform, it's definitely not chrome (see caniuse).